language/move-lang/tests/move_check/parser/address_too_long.exp

```diff
@@ -3,6 +3,6 @@ error:
    ┌── tests/move_check/parser/address_too_long.move:2:9 ───
    │
  2 │ address 0x112233445566778899101122334455667788992011223344556677889930112233:
-   │         ^ The address [17, 34, 51, 68, 85, 102, 119, 136, 153, 16, 17, 34, 51, 68, 85, 102, 119, 136, 153, 32, 17, 34, 51, 68, 85, 102, 119, 136, 153, 48, 17, 34, 51] is of invalid length. Addresses are at most 32-bytes long
+   │         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ The address [17, 34, 51, 68, 85, 102, 119, 136, 153, 16, 17, 34, 51, 68, 85, 102, 119, 136, 153, 32, 17, 34, 51, 68, 85, 102, 119, 136, 153, 48, 17, 34, 51] is of invalid length. Addresses are at most 32-bytes long
```

Comment:
Maybe not needed in this PR since it already existed, but is there any better way of printing out the address? This seems a bit nasty of an error message.

Reply:
If this message is only reported from the parser (@tnowacki is it?), I suggest we change it to: "Address is longer than the maximum size of 32 bytes" (which is hopefully shrinking to 16 soon). Then we don't need to repeat the address value in the error message. WDYT?

Reply:
Yeah its only for the parser. "Address is longer than the maximum size of 32 bytes" seems like an ok error message, no need to dump out the value

Reply:
I'll make a separate PR for that change. Is this OK to land then?

Reply:
This sounds good to me. I'm OK with landing this as-is.